sensors/nxt_i2c_sensor_defs.c

```diff
@@ -262,6 +262,19 @@ static int ht_angle_scale(void *context,
 	return 0;
 }
 
+static int ms_pps58_nx_set_mode_post_cb(nxt_i2c_sensor_data *sensor, u8 mode)
```

Comment:
please add a comment above to separate this from other sensor types. And it would be good to explain what we are doing here too.